bleak/backends/bluezdbus/client.py

```diff
@@ -605,17 +605,17 @@ async def write_gatt_descriptor(self, handle: int, data: bytearray) -> None:
     async def start_notify(
         self,
         char_specifier: Union[BleakGATTCharacteristicBlueZDBus, int, str, uuid.UUID],
-        callback: Callable[[str, Any], Any],
+        callback: Callable[[int, bytearray], None],
         **kwargs
     ) -> None:
         """Activate notifications/indications on a characteristic.
 
-        Callbacks must accept two inputs. The first will be a uuid string
-        object and the second will be a bytearray.
+        Callbacks must accept two inputs. The first will be a integer handle of the characteristic generating the
+        data and the second will be a ``bytearray``.
```

Comment:
Would be nice to say that the bytearray contains the data itself.

Reply:
same comment applies to other backends

Reply:
Also a good idea.